features/gezag-minderjarige/3.1-recente-gebeurtenis-ontkenning.feature

```diff
@@ -0,0 +1,117 @@
+  | 1.4 Uitspraak gezag aanwezig?                                          | Ja                                                                                 |
+
+  Het is mogelijk dat er een uitspraak is gedaan door de rechter, maar door een recentere gebeurtenis het gezag toch weer van rechtswege is verkregen of ingetrokken (ontkenning). 
+  Dit is (mogelijk) het geval bij de volgende gebeurtenissen: adoptie, hertrouwen of ontkenning ouderschap.
+
+  Ontkenning ouderschap
+  Omvat: ontkenning vaderschap (art. 1:200 en 1:201 BW) of moederschap (art. 1:202a en 1:202b BW) of nietige erkenning (art. 1:204) of vernietiging erkenning (art.
+    1:205 en 1:205a BW).  
+  Het door het huwelijk of geregistreerd partnerschap ontstane ouderschap wordt geacht nimmer gevolg te hebben gehad (art. (1:202b in
+  verbinding met) art. 1:202 lid 1 BW). Door de nietigheid/vernietiging van de erkenning wordt de erkenning geacht nimmer gevolg te hebben gehad 
+  (art. 1:206 lid 1 BW). Na ontekenning is de ouder niet meer juridisch ouder van het kind, terwijl de ouder volgens een eerdere uitspraak wel het gezag had.
+
+  Hoe te achterhalen?
+  Ontkenning ouderschap/vernietiging erkenning: 
+  Indicatie gezag bevat de waarde ‘1’ of ‘12’ en de geslachtsnaam van die ouder komt niet voor of bevat de waarde ‘.’. of de indicatie gezag bevat de waarde ‘2’ of ‘12’ 
+  en de geslachtsnaam van die ouder komt niet voor of bevat de waarde ‘.’.
+  Een ontkenning ouderschap en/of een vernietiging van een erkenning hebben altijd terugwerkende kracht en er ontstaat een ‘lege’ oudercategorie,
+  herkenbaar aan het ontbreken van de geslachtsnaam. Als de geslachtsnaam wel voorkomt maar de waarde ‘.’ bevat dan is (nog) niet bekend wie die ouder is.
+ 
+  Gebruikte velden:
+    - Ouder -> 02.02.40
+    - Geslachtsnaam van de ouder -> 03.02.40
+    - Indicatie gezag -> 11.32.10
+    - Ingangsdatum gezag -> 11.85.10
+    - Datum ingang familierechtelijke betrekking -> 02.62.10 en 03.62.10
+
+    Achtergrond:
+      Gegeven de persoon 'Nelly' met burgerservicenummer '000000012'
+      * is meerderjarig, niet overleden en staat niet onder curatele
+      En de persoon 'Johnnie' met burgerservicenummer '000000024'
+      * is meerderjarig, niet overleden en staat niet onder curatele
+      En 'Nelly' en 'Johnnie' zijn met elkaar gehuwd
+      En de persoon 'Kees' met burgerservicenummer '000000036'
+      * is ingeschreven in de BRP
+      * is minderjarig
+      * is niet geëmigreerd geweest
+      * is in Nederland geboren
+      * heeft 'Nelly' als ouder 1
+      * heeft 'Johnnie' als ouder 2
+
+  Regel: Als vaderschap is ontkend of de erkenning is vernietigd en er was een uitspraak gezag voor die ouder, wordt het gezag van rechtswege bepaald
+    
+    Abstract Scenario: er is uitspraak gezag voor ouder 1 en ouder 2 en erkenning door ouder <ouder> is ontkend of vernietigd er is sprake van EenhoofdigOuderlijkGezag
```

Comment:
Omdat het nu lastig te controleren is omdat er features zijn die nu falen in 3.1-recente-gebeurtenis-ontkenning zouden we eventueel zolang de check op IsStaandeHuwelijkOfPartnerschapGeboren nog niet aangepast is, deze scenario's kunnen taggen met @skip-verify.

```suggestion
    @skip-verify
```